plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProvider.java

```diff
@@ -319,7 +321,8 @@ private Instance createInstance(String imageId,
                 volumes.addAll(devMachineSystemVolumes);
                 volumes.addAll(systemVolumesForMachine);
 
-                env = new String[] { apiEndpointEnvVariable };
+                env = new String[] { apiEndpointEnvVariable,
+                                     DockerInstanceMetadata.PROJECTS_ROOT_VARIABLE + "=" + PROJECTS_FOLDER_PATH };
```

Comment:
Character literal <code>'='</code>  should be used rather than <code>"="</code> string literal in concatenation.